java/google/registry/rde/imports/RdeImportUtils.java

```diff
@@ -66,10 +83,11 @@ public RdeImportUtils(
     this.escrowBucketName = escrowBucketName;
   }
 
-  private <T extends EppResource> void importEppResource(final T resource, final String type) {
+  public <T extends EppResource> ImmutableSet<Object> getIndexesForEppResource(T resource,
```

Comment:
If I do that, I just need to convert the set later on to use Object. Instances of TransferServerApproveEntity are added to this list later on if a pending transfer is in progress. Changing this from Object to ImmutableObject will complicate code elsewhere and make it less readable.